src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -76,6 +82,9 @@ export const QueryAssistSummary: React.FC<QueryAssistSummaryProps> = (props) =>
     defaultMessage:
       'Thank you for the feedback. Try again by adjusting your question so that I have the opportunity to better assist you.',
   });
+  const errorPrompt = i18n.translate('queryEnhancements.queryAssist.summary.errorPrompt', {
```

Comment:
should we consider adding : `Try again by adjusting your question so that I have the opportunity to better assist you.`

Or do we have an error message that we can display to the user for more insight?

Reply:
Some errors are not caused by questions, it might be frustrating when there's backend issue but we keep asking users to change questions.